packages/nvidia_nat_core/src/nat/runtime/user_manager.py

```
@@ -0,0 +1,243 @@
+        """
+        auth: str | None = None
+
+        if isinstance(connection, Request):
+            auth = connection.headers.get("authorization")
+        elif isinstance(connection, WebSocket) and hasattr(connection, "scope") and "headers" in connection.scope:
+            for name, value in connection.scope.get("headers", []):
+                try:
+                    name_str: str = name.decode("utf-8").lower()
+                    value_str: str = value.decode("utf-8")
+                except Exception:
+                    continue
```

Ruff (0.15.5)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)